README.md

```diff
@@ -1,2 +1,57 @@
-# openmrs-module-tag
-Provides a generic tagging mechanism for all domain objects
+# Generic Tagging Module
+This Module provides a generic mechanism that allows users to annotate domain objects with simple text labels/tags, 
+#### Table of Contents
+
+1. [Build](#build)
+   1. [Prerequisites](#prerequisites)
+   2. [Build Command](#build-command)
+   3. [Documentation](#documentation)
+
+## Build
+### Prerequisites
+
+#### Java
+
+The module is a Java application which is why you need to install a Java JDK.
+
+If you want to build the master branch you will need Java 7.
```

Comment:
Are you sure you need java 7? Or did you just paste and commit? Please read through the entire file to make sure everything applies

Reply:
I didn't use anything that requires Java 8

Reply:
I'm not saying require java 8, the project where you copied this requires java 7, are you sure your module requires java 7, please read the file, cross check and make sure everything applies.

Reply:
Sorry, I'll probably have to test this properly.

Reply:
The question is what have you been using? And anyways you should require java 6

Reply:
Can you please address this?